sev/src/lib.rs

```diff
@@ -12,10 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![warn(clippy::all)]
+#![deny(clippy::all)]
 #![allow(unknown_lints)]
 #![allow(clippy::identity_op)]
 #![allow(clippy::unreadable_literal)]
+#![allow(clippy::uninit_assumed_init)]
```

Comment:
Clippy gets angry about this because it is potential UB. However I'm not sure how to fix this, it looks like the lack of initialization is intentional, but is also UB.

Reply:
I would selectively add `#[allow(clippy::uninit_assumed_init)]` to the blocks in question rather than adding it here.